### PR TITLE
Update to use the latest Angular/pkg:angular_test

### PR DESCRIPTION
**Pull request by matanlurey**
Actually closes https://github.com/dart-lang/angular_test/issues/16.

Also closes https://github.com/dart-lang/angular2_api_examples/pull/3

### CODE REVIEW

File: testing/pubspec.yaml [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+  pageloader: '^2.2.5'
+  test: '0.12.18+1'
+
+# These are required in order to use AngularDart 3.0.0-alpha with ng_bootstrap.
 dependency_overrides:
-  angular2:
-    git: 
-      url: https://github.com/dart-lang/angular2
-      ref: 8cd7f35d74a9a8e323d6f0cf73ddda5acb079fe3
+  angular2: '3.0.0-alpha'
```
Comment (kevmoo) [inline testing/pubspec.yaml:23]: This seem redundant w/ a lot of what's in dependencies above, no?

File: testing/pubspec.yaml [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+  intl: '0.14.0'
+  markdown: '0.11.2'
+  reflectable: any
```
Comment (kevmoo) [inline testing/pubspec.yaml:26]: Why reflectable here?

**Review by kevmoo — APPROVED** [review]
Looks okay, although I'd like to understand how this lines up with docs we publish – since this references yet-to-be-published packages.

Are you working with @kwalrath et el?

File: testing/pubspec.yaml (hunk 1 above)
Comment (matanlurey) [reply testing/pubspec.yaml:23]: I needed this temporarily to get ng_bootstrap working again.

File: testing/pubspec.yaml (hunk 2 above)
Comment (matanlurey) [reply testing/pubspec.yaml:26]: I needed this temporarily to get ng_bootstrap working again.

**Comment by matanlurey** [comment]
@kevmoo I haven't shown this to @kwalrath yet - I want to make sure the API is mature and is working before I have any documentation formally generated. I'll update her soon.

**Pull request merged**